### DISCUSSION TITLE
자바스크립트에서 일어나는 데이터 형변환에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#31] [타입변환과 단축 평가]
- 자바스크립트에서 일어나는 데이터 형변환에 대해 설명해주세요.

**Comment by pipisebastian** [comment]
>자바스크립트에서 일어나는 데이터 형변환에 대해 설명해주세요.

자바스크립트에서 형변환은 2가지로 나눌 수 있습니다. 명시적 타입변환과 암묵적 타입변환인데요,
명시적 타입변환은 개발자가 의도적으로 값의 타입을 변환하지만, 암묵적은 의도와 상관없이 자동으로 변환됩니다.

**Comment by publdaze** [comment]
> - 자바스크립트에서 일어나는 데이터 형변환에 대해 설명해주세요.
- 명시적 타입 변환과 암묵적 타입 변환으로 나눌 수 있음.
- 명시적 타입 변환은 의도적으로 타입을 변환하지만, 암묵적 타입 변환은 의도하지 않은 상태에서 타입이 변환 되는 걸 말함
